developer/src/kmcmpdll/kmcmpdll.vcxproj.filters

```diff
@@ -88,6 +67,23 @@
     <ClInclude Include="CheckFilenameConsistency.h">
       <Filter>Header files</Filter>
     </ClInclude>
+    <ClInclude Include="$(KEYMAN_ROOT)\developer\src\ext\json-schema-validator\json-schema.hpp" />
+    <ClInclude Include="$(KEYMAN_ROOT)\developer\src\ext\json\json.hpp" />
```

Comment:
No header files?

Reply:
Visual Studio moved these out I think? I didn't update this file manually.

Fixed.